test/onnx/test_fx_op_consistency.py

```diff
@@ -190,6 +190,11 @@ def skip_torchlib_forward_compatibility(
         dtypes=onnx_test_common.COMPLEX_TYPES,
         reason=onnx_test_common.reason_dynamo_does_not_support("Addr", "complex64")
     ),
+    xfail(
+        "all",
+        reason=onnx_test_common.reason_onnx_does_not_support(
+            "Op (ReduceXXX) [ShapeInferenceError] axis must be in [-rank, rank-1]. input rank was 0")
```

Comment:
cc @justinchuby

Reply:
Why is this suddenly popped out on ORT==1.17?

Reply:
need investigation whether it's export bug or ort regression.

Reply:
https://github.com/pytorch/pytorch/issues/119441